pkg/sfu/peer.go

```diff
@@ -30,6 +31,16 @@ type SessionProvider interface {
 	GetSession(sid string) (*Session, WebRTCTransportConfig)
 }
 
+const (
```

Comment:
These might be better defined in the subscriberapi.go, not a strong opinion on this though just something to think about

Reply:
I've been debating the same. Since the method name can be anything, we could put constants in the appropriate location (session and subscribeapi.go), that way middleware would be on one place, but on the other hand this way all available methods are in one place. Not a strong opinion either.